examples/reflection/trait_reflection.rs

```diff
@@ -1,3 +1,5 @@
+//! Allows reflection with trait objects
```

Comment:
This isn't terribly descriptive; but IMO we should tackle this as part of a push to properly document and explain bevy_reflect instead.